ui/app/mainui/Popups.qml

```diff
@@ -0,0 +1,139 @@
+import QtQuick 2.14
+
+import AppLayouts.Chat.popups 1.0
+import shared.popups 1.0
+
+import utils 1.0
+
+QtObject {
+    id: root
+
+    /* required */ property QtObject popupsParent
```

Comment:
We had a bug previously about this, so:
```suggestion
    /* required */ property var popupsParent
```

Reply:
This property here was a leftover. Removed completely :)